src/helpers/authManager.js

```diff
@@ -111,7 +111,7 @@ const initOAuth = (bInit) => {
         : `${window.location.origin}${window.location.pathname}`,
     authService: sdkConfigAuth.authService,
     appAlias: sdkConfigAuth.appAlias || '',
-    useLocking: true
+    useLocking: true,
```

Comment:
Don't think this line required modification.  Not sure why trailing comma was added (perhaps no harm)? Not sure how lint rules deal with this whether they suggest you add that or not add that.

Reply:
Removed this comma.